### ISSUE TITLE
The NuGet package incorrectly references the Pipeline DLL when targeting .NET 4.0

### ISSUE THREAD

**Issue by craftworkgames** [craftworkgames/MonoGame.Extended#37]
The NuGet package contains 2 DLLs.

 - MonoGame.Extended.dll
 - MonoGame.Extedded.Content.Pipeline.dll

The intention is that the `MonoGame.Extended.dll` is referenced by the game and the `MonoGame.Extended.Content.Pipeline.dll` is referenced by the MonoGame Pipeline tool. If your game is targeting .NET 4.5 this works as intended. Only the first DLL is referenced by the game and the second DLL is left in the packages folder ready to be referenced by the Pipeline.

This morning I discovered that if the game project is targeting .NET 4.0 (or anything other than 4.5) it will incorrectly reference both DLLs in the game project.

The problem seems to be the way the nuspec file is configured.

    <references>
      <group targetFramework="portable-net45+sl50+MonoAndroid10+xamarinios10+MonoTouch10">
        <reference file="MonoGame.Extended.dll" />
      </group>
    </references>

Maybe we need to remove the target framework from the group? Or make more groups for each different target? Requires more [investigation](https://docs.nuget.org/create/nuspec-reference).

**Comment by craftworkgames** [comment]
I started looking into this today and found that the issue runs a little deeper than the nuspec file. It turns out that the core library also references the .NET 4.5 version of Newtonsoft.Json which prevents a .NET 4 application referencing the library.

The core library shouldn't really carry a dependency on anything other than MonoGame itself so I'm going to do a bit of a refactor on the BitmapFont code to sort this out. The BitmapFont code was added to the library before I knew how to do the Content Pipeline stuff properly, so it left around a bit of technical debt.
